app/src/main/java/co/tinode/tindroid/db/MessageDb.java

```diff
@@ -61,9 +64,18 @@
      */
     private static final String COLUMN_NAME_TS = "ts";
     /**
-     * Server-issued sequence ID, integer, indexed
+     * Server-issued sequence ID, integer, indexed. If the message represents
+     * a deleted range, then <tt>seq</tt> is the lowest bound of the range.
      */
     private static final String COLUMN_NAME_SEQ = "seq";
+    /**
+     * If message represents a deleted range, this is the upper bound of the range.
```

Comment:
Can you also mention the interval is open on the right please?

Reply:
Done